pkgs/development/tools/efm-langserver/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "efm-langserver";
+  version = "0.0.29";
+  src = fetchFromGitHub {
+    owner = "mattn";
+    repo = "efm-langserver";
+    rev = "v${version}";
+    sha256 = "sha256-/G3GF4tz0MzR74aLKViKm03jT8e1MOzAjSpq9Aof5n8=";
+  };
+
+  modSha256 = vendorSha256;
+  vendorSha256 = "sha256-tca+1SRrFyvU8ttHmfMFiGXd1A8rQSEWm1Mc2qp0EfI=";
```

only vendorSha is used for buildGoModule https://nixos.org/manual/nixpkgs/stable/#ssec-go-modules
```suggestion
  vendorSha256 = "sha256-tca+1SRrFyvU8ttHmfMFiGXd1A8rQSEWm1Mc2qp0EfI=";
```